src/translations.php

```diff
@@ -138,6 +138,14 @@
         "Longest Week Streak" => "pi long tras semèn",
         "Present" => "Prezan",
     ],
+    "hy" => [
+        "Total Contributions" => "Ընդհանուր ներդրումը",
+        "Current Streak" => "Ընթացիկ շարք",
+        "Longest Streak" => "Ամենաերկար շարք",
+        "Week Streak" => "Ընթացիկ շաբաթների շարք",
+        "Longest Week Streak" => "Ամենաերկար շաբաթների շարք",
+        "Present" => "Ներկայումս",
```

Comment:
Similarly, this word is rather long. We could add a linebreak, or possibly choose a similar word that is shorter. I do not know the language, so I am asking for your opinion on this. Do you think "Այսօր" would be an acceptable alternative?

```suggestion
        "Present" => "Այսօր",
```

Reply:
"Այսօր" can't be used because in Armenian it means "Today", not "Present". So, here's some similar and close words to "Ներկայումս":

```suggestion
        "Present" => "Այժմ",
```

or

```suggestion
        "Present" => "Հիմա",
```

---

These tho words actually much more means something like "now". Which version is better in your opinion?

Reply:
Words like "today" and "now" are also fine and in some languages it's the same word. For example in Hebrew we have היום/*hayom* which means "today" or in Japanese, 今/*ima* which means "now". The context is that it's used at the end of a date range (eg. 2023/03/26 - Present) where "present" represents the current date. Whatever word is most natural there for representing "current date"/"present"/"today"/"now"/etc should be used. I don't know what that would be in Armenian, but copilot (gpt) suggested "Այսօր" but of course it's not always correct, so I'd rather let you decide which of the words, including the ones you suggested, fits the context most naturally.

Reply:
In my opinion using "Այժմ" instead of "Այսօր" is more correct, so I'd choose that. I'll commit that version